.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I don't use vscode, but is it considered normal to check these files in? If yes, disregard my remark.

Reply:
I believe so. The workspace settings file should contain only settings which are relevant to everyone using the project. There's a separate user settings file for personal preference settings.

Here I've included settings to display a ruler at the 120 character mark, to disable automatically running jest tests (because otherwise you'll constantly be creating and deleting buckets in the background), and to use the version of TypeScript that's in the project dependencies. I think these are pretty widely applicable. Workspace settings would also be a good spot to configure identation.

Certainly in private projects (at companies where everyone uses the same IDE) it's normal to include these. But in public projects that are IDE-agnostic it's less common. Anyway, including it doesn't do any harm.